README.md

```diff
@@ -61,13 +57,21 @@ applications.
 ## API
 
 The library exports the following properties:
+- `CONTEXT_URL`
+- `CONTEXT`
```

Comment:
Do we need to say anything about packages that bundle more than one context?

Reply:
@dlongley as in like "don't do it"?

Reply:
@dmitrizagidulin, yes, that's fine advice. I thought we were doing it somewhere but was mistaken.

Reply:
(Added a comment.)